### PR TITLE
ETC 9500 - 12100 check

### PR DESCRIPTION
**Pull request by vtange**
Continuing where I left off
stopping at 12100 since from here on it's obviously quests of Cursed City
EDIT: oooh best PR# ever

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -9770,11 +9770,11 @@ ETC_20150323_009771	{memo X}This monster is enslaved!
 ETC_20150323_009772	Incapacitated
 ETC_20150323_009773	Incapacitated by Evening Star Rune.
 ETC_20150323_009774	Possession
-ETC_20150323_009775	Hauberk is possessed by this monster.
+ETC_20150323_009775	This monster holds part of Hauberk.
```
Comment (Sunaries) [inline ETC.tsv]: holds **a** part of?

File: ETC.tsv [hunk 2]
```diff
@@ -12066,16 +12066,16 @@ ETC_20150714_012067	Normal Husband Down
...
-ETC_20150714_012077	There is a player of similar level nearby.{nl}You could invite this player to the party.
-ETC_20150714_012078	This party of similar level has requested that you join them. Join this party! 
-ETC_20150714_012079	You've found a fellow player who you've played with before. {nl}You could invite this player to your party again!
-ETC_20150714_012080	Players who you've played with before are inviting you to their party again. Join this party again!
+ETC_20150714_012075	A player who is doing the same quest is nearby. Invite them to the party!
+ETC_20150714_012076	A party doing the same quest as you is requesting your participation. Join the party!
+ETC_20150714_012077	There is a player of similar level nearby.{nl}You could invite this player to your party.
+ETC_20150714_012078	A party of similar level to you has requested that you join them. Join this party! 
+ETC_20150714_012079	You've found a fellow player whom you've played with before. {nl}You could invite this player to your party again!
+ETC_20150714_012080	A player whom you've played with before are inviting you to their party again. Join this party again!
```
Comment (Sunaries) [inline ETC.tsv]: are inviting = is inviting

File: ETC.tsv [hunk 3]
```diff
@@ -12066,16 +12066,16 @@ ETC_20150714_012067	Normal Husband Down
...
-ETC_20150714_012075	A player who is doing the same quest is near. Invite them to the party!
-ETC_20150714_012076	This party that is doing the same quest is requesting your participation. Join the party!
-ETC_20150714_012077	There is a player of similar level nearby.{nl}You could invite this player to the party.
-ETC_20150714_012078	This party of similar level has requested that you join them. Join this party! 
-ETC_20150714_012079	You've found a fellow player who you've played with before. {nl}You could invite this player to your party again!
-ETC_20150714_012080	Players who you've played with before are inviting you to their party again. Join this party again!
+ETC_20150714_012075	A player who is doing the same quest is nearby. Invite them to the party!
+ETC_20150714_012076	A party doing the same quest as you is requesting your participation. Join the party!
+ETC_20150714_012077	There is a player of similar level nearby.{nl}You could invite this player to your party.
+ETC_20150714_012078	A party of similar level to you has requested that you join them. Join this party!
```
Comment (Sunaries) [inline ETC.tsv]: of similar = of a similar

File: ETC.tsv [hunk 4]
```diff
@@ -12066,16 +12066,16 @@ ETC_20150714_012067	Normal Husband Down
...
 ETC_20150714_012074	There's a player who seems to be fit for your party.{nl}You could invite this person to your party!
-ETC_20150714_012075	A player who is doing the same quest is near. Invite them to the party!
-ETC_20150714_012076	This party that is doing the same quest is requesting your participation. Join the party!
-ETC_20150714_012077	There is a player of similar level nearby.{nl}You could invite this player to the party.
-ETC_20150714_012078	This party of similar level has requested that you join them. Join this party! 
-ETC_20150714_012079	You've found a fellow player who you've played with before. {nl}You could invite this player to your party again!
-ETC_20150714_012080	Players who you've played with before are inviting you to their party again. Join this party again!
+ETC_20150714_012075	A player who is doing the same quest is nearby. Invite them to the party!
+ETC_20150714_012076	A party doing the same quest as you is requesting your participation. Join the party!
+ETC_20150714_012077	There is a player of similar level nearby.{nl}You could invite this player to your party.
```
Comment (Sunaries) [inline ETC.tsv]: of similar = of a similar

**Comment by vtange** [comment]
Good to Merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**